tool/build-robots-txt.js

```diff
@@ -31,7 +31,7 @@ async function runBuildRobotsTxt(outfile) {
       content += `Disallow: /${locale}/search\n`;
     }
   }
-  fs.writeFileSync(outfile, `${content.trim()}\n`, "utf-8");
+  fs.writeFileSync(outfile, `${content.trim()}\n`, "utf8");
```

Comment:
For `fs.writeFileSync()` we may be able to omit it altogether, since `utf8` is already the default (but not for `readFileSync`!):

https://nodejs.org/dist/latest-v16.x/docs/api/fs.html#fswritefilesyncfile-data-options

```suggestion
  fs.writeFileSync(outfile, `${content.trim()}\n`);
```